gson/pom.xml

```diff
@@ -163,6 +200,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
```

Comment:
Can we keep java 7 ? Is there any reason to make it java 8 ? 

I'm not sure if this module is being used by Android developers.

Reply:
Sure